lib/inferno/apps/cli/templates/lib/%library_name%/metadata.rb.tt

```diff
@@ -0,0 +1,18 @@
+require_relative 'version'
+
+module <%= module_name %>
+  class Metadata < Inferno::TestKit
+    id :<%= test_kit_id %>
+    title '<%= title_name %>'
+    description <<~DESCRIPTION
+      This is a big markdown description of the test kit.
+    DESCRIPTION
+    suite_ids [:<%= test_suite_id %>]
+    # tags ['SMART App Launch', 'US Core']
+    # last_updated '2024-03-07'
+    version VERSION
+    maturity 'Low'
+    authors <%= authors %>
```

Comment:
If authors is blank this doesn't just leave a blank authors line?  Should anything in here that is optional in the cli work fine without a check?

Reply:
If you don't supply an author, it defaults to your username. You can look at what's generated after running `inferno new`.